package.json

```diff
@@ -54,10 +54,7 @@
     }
   },
   "lint-staged": {
-    "*.js": [
-      "npm run fixlint",
-      "git add"
-    ]
+    "*.js": "eslint --fix"
```

Comment:
lets keep the `fixlint` can change the script if you want to do any changes.

Reply:
Since we're specifying target files with a glob pattern it would be kinda redundant to have `fixlint` instead.

Reply:
I guess you can change the `fixlint` by removing the `.` .
but I forgot lint-stage does check only the staged file, and not sure `eslint --fix` and `eslint . --fix` will lint the same files or not